### PR TITLE
US106116-Update activity-usage to use new sdk entity

### PR DESCRIPTION
**Pull request by edwardg8**

### CODE REVIEW

File: EnrollmentEntity.js [hunk 1]
```diff
@@ -19,10 +19,44 @@ export class EnrollmentEntity extends Entity {
...
+
+	dueDate() {
+		if (!this._entity) {
+			return;
+		}
+		return this._sirenClassProperty(this._entity, 'due-date');
+	}
+
+	isAttended() {
+		return this._entity && this._entity.hasClass('attended');
```
Comment (m6jones) [inline EnrollmentEntity.js]: Should we make `'attended'` a constant in the `EnrollmentEntity` 

Unless you think doing that is not a good idea. The idea was that we could document possible classes in the entity file so the file it self is sort of documentation on what the entity can contain.

Comment (m6jones) [reply EnrollmentEntity.js]: Didn't notice which file we were in.

Comment (m6jones) [reply EnrollmentEntity.js]: You can do `this.hasClass('attended')` since the parent has that method.

File: EnrollmentEntity.js [hunk 2]
```diff
@@ -19,10 +19,44 @@ export class EnrollmentEntity extends Entity {
...
+	}
+
 	onOrganizationChange(onChange) {
 		const organizationHref = this.organizationHref();
 		// _subEntity builds new sub entity and allows this object to track it.
 		// So all sub entities are dispose when this object is disposed.
 		organizationHref && this._subEntity(OrganizationEntity, organizationHref, onChange);
 	}
+
+	_sirenClassProperty(entity, sirenClass) {
```
Comment (m6jones) [inline EnrollmentEntity.js]: This is cool!!!

File: EnrollmentEntity.js (hunk 1 above)
Comment (edwardg8) [reply EnrollmentEntity.js]: Oh see it now, that's cool, will change this.

**Pull request merged**